src/sgame/components/TurretComponent.cpp

```diff
@@ -275,20 +277,20 @@ Vec3 TurretComponent::RelativeAnglesToAbsoluteAngles(const Vec3 relativeAngles)
 	QuatToAngles(absoluteRotation, absoluteAngles);
 
 	/*turretLogger.Debug("RelativeAnglesToAbsoluteAngles: %s → %s. Torso angles: %s.",
-		Utility::Print(relativeAngles), Utility::Print(Vec3::Load(absoluteAngles)), TorsoAngles()
+		Utility::Print(relativeAngles), Utility::Print(glm::vec3::Load(absoluteAngles)), TorsoAngles()
```

Comment:
This isn't used. I guess you intended this:
```suggestion
		Utility::Print(relativeAngles), Utility::Print(VEC2GLM(absoluteAngles)), TorsoAngles()
```
but this code is commented and there is probably no `Utility::Print(glm::vec3);`

Reply:
Deleted the commented-out code.